### ISSUE TITLE
Support a banner feature to communicate what performance can be expected from remote server

### ISSUE THREAD

**Issue by dranch** [esnet/iperf#283]
Network performance over a WAN is completely dependent on it's last hop.  It would be helpful that when an iperf client connects to an iperf server, the server can display a short banner on what the performance should be expected from it (it's raw link speed, expected thoughput, PPS, latency, any QoDS policies, rules for it's use, whatever).

**Comment by bmah888** [comment]
That first sentence is not universally true.  Bottlenecks can happen at any step along a path, whether wide-area or local-area.  Given that, I'm trying to understand the utility of this message; the remote server can't always determine the expected performance.

**Comment by dranch** [comment]
I would agree that it's not universally true but with a banner, the user can at least get an idea of what the best possible performance might be.  Personally, I think allowing for a general MOTD like text file to be displayed could offer many helpful types of communications about the iperf server.
